docs/learning-the-basics.md

```diff
@@ -11,7 +11,16 @@ At the top of the screen you can see the Panel. On the left is the Applications
 ![The Panel](../images/docs/learning-the-basics/panel.png)
 
 #### Applications {#applications}
-On the left side of the panel is the Applications item. Clicking **Applications** brings up a launcher with all of your installed apps. You can view multiple pages of apps using the pagers at the bottom or by scrolling. You can also use the view switcher at the top to switch between a grid view and a category view. Lastly, you can search the launcher by typing at any time.
+On the left side of the panel is the Applications item. Clicking **Applications** brings up a launcher with all of your installed apps. You can use the view switcher at the top to switch between a grid view and a category view. You can view multiple pages of apps using the pagers at the bottom or by scrolling.
+
+You can search for apps by name or by keyword and perform actions associated with them. You can also search for system settings panes. Some of the actions you can find in search include:
+
+* Shutdown and Restart the computer
+* Compose a new email
+* Open a Private Browsing window
+* Math operations (example: 23 * 5)
```

Comment:
Exact commands for these are:
`Shut Down` and `Restart`
`Compose Message`
`New Private Browsing Window`
`23 * 5`